Libplanet/Blockchain/BlockLocator.cs

```diff
@@ -8,6 +9,37 @@ internal class BlockLocator : IEnumerable<HashDigest<SHA256>>
     {
         private readonly IEnumerable<HashDigest<SHA256>> _impl;
 
+        public BlockLocator(
+            Func<long, HashDigest<SHA256>?> indexBlockHash,
```

Comment:
```suggestion
            Func<long, HashDigest<SHA256>?> blockHashByIndex,
```

Reply:
I believe current naming is more intuitive, because it's a function and natural to read it as “to index a block hash.”  Ditto for the same reviews below.